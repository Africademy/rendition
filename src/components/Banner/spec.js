/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import { Banner, Provider } from '../../../dist'

test('Banner renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Banner color='white'>
        <h1>balena</h1>
        <p>
          Balena brings the benefits of Linux containers to the IoT. Develop
          iteratively, deploy safely, and manage at scale.
        </p>
      </Banner>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
