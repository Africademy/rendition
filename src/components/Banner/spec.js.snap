// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
.c0 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
  line-height: 1.5;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  min-height: 80vh;
  background-size: cover;
  background-position: center;
  background-image: none;
}

.c3 {
  padding: 16px;
  color: white;
}

.c1 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
}

.c1 h1,
.c1 h2,
.c1 h3,
.c1 h4,
.c1 h5,
.c1 h6,
.c1 button {
  font-family: CircularStd,Arial,sans-serif;
}

@media screen and (min-width:576px) {
  .c3 {
    padding: 38px;
  }
}

<div
  className="c0 c1"
>
  <div
    className="c2 c3"
  >
    <h1>
      balena
    </h1>
    <p>
      Balena brings the benefits of Linux containers to the IoT. Develop iteratively, deploy safely, and manage at scale.
    </p>
  </div>
</div>
`;
